.github/workflows/fuzzers.yml

```diff
@@ -37,7 +37,6 @@ jobs:
           - honggfuzz_qemu
           - weizz_qemu
           # Temporary variants.
-          - aflcc_no_orig_dict
```

Comment:
Don't understand what this removal is doing here.

Reply:
aflcc does not work across a lot of benchmarks [atleast 4-5]. While aflcc is explicitly excluded, this variant is not. I prefer to keep this temporary variant out of CI, rather than adding more blacklist for this temp variant.